branding/README.md

```diff
@@ -14,4 +14,17 @@ Branding definition file has the following variables
 * `MSI_PRODUCTCODE`: Windows installer uses UUID to identify which MSI files are of the same lineage.
   If two MSIs have the same UUID, one will overwrite another. So if you are to produce your own MSI,
   you need to use a different UUID.
-* `OSX_IDPREFIX`: prefix of the various IDs in OS X. This follows the reverse domain name format.
\ No newline at end of file
+* `OSX_IDPREFIX`: prefix of the various IDs in OS X. This follows the reverse domain name format.
+* `AUTHOR`: author line for software
+* `LICENSE`: applicable license
+* `HOMEPAGE`: homepage for this distribution
+* `CHANGELOG_PAGE`: web page for the changelog
```

Comment:
I'd appreciate bit more explanation of what those variables are. The idea is that somebody else should be able to write these files without hunting down where they are actually used. So the description should include the expected format and length, an example, and that sort of things.

That'll also help people decide when to reuse existing variables and when to introduce new ones.